### PR TITLE
Install

### PR DESCRIPTION
**Pull request by rpspringuel**
These updates to the Mac and Windows installers (and uninstallers) get them to reflect the texmf structure that was implemented in `install-gtex.sh` at the suggestion of the TeXLive people.

This also addresses #1081 by having removing the `*.sty` files from their old location before installing.  The removal instructions are structures to also get rid of the parmesan font files.

### CODE REVIEW

**Comment by rpspringuel** [comment]
I realize that no one else can test what I did for the Mac and Windows installers, but if some one would take a look at the changes to `install-gtex.sh` before I merge this, I'd appreciate it.

File: install-gtex.sh [hunk 1]
```diff
@@ -67,17 +67,27 @@ TTFFILES=(gregorio.ttf greciliae.ttf granapadano.ttf gregorio-op.ttf
           gresgmodern.ttf)
 DOCFILES=(doc/Appendix*.tex doc/Command*.tex doc/Gabc.tex
           doc/*Ref.tex doc/*Ref.lua doc/*.gabc doc/Gregorio*Ref-$FILEVERSION.pdf)
-EXAMPLEFILES=(examples/FactusEst.gabc examples/PopulusSion.gabc examples/main-lualatex.tex)
+EXAMPLEFILES=(examples/FactusEst.gabc examples/PopulusSion.gabc examples/main-lualatex.tex examples/debugging.tex)
```
Comment (henryso) [inline install-gtex.sh]: This line is a little long.  I would put the last bit on the next line.

**Comment by henryso** [comment]
All I had was a really, really minor comment.  `install-gtex.sh` looks fine.

**Comment by rpspringuel** [comment]
Thanks for the look-see.  I've made the changes.

**Pull request merged**